src/makeCalculator.js

```diff
@@ -37,7 +37,40 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(num) {
+      this.result += num;
+
+      return this;
+    },
```

Comment:
By the condition of the task, only operate and reset must return this

```suggestion
    add(num) {
      this.result += num;
    },
```